AUTHORS.rst

```diff
@@ -1,64 +1,36 @@
-- Alper Cinar <alper@srdc.com.tr>
-- Artem Tsikiridis <artem.tsikiridis@cern.ch>
-- Benoit Thiell <bthiell@cfa.harvard.edu>
-- Björn Oltmanns <bjoern.oltmanns@gmail.com>
-- Carmen Alvarez Perez <carmen.alvarez.perez@cern.ch>
-- Christopher Hayward <christopher.james.hayward@cern.ch>
-- Christopher Parker <chris.parker.za@gmail.com>
-- Cristian Bacchi <cristian.bacchi@gmail.com>
-- Diane Berkovits <diane.berkovits@cern.ch>
-- Eirini Psallida <eirini.psallida@cern.ch>
-- Esteban J. G. Gabancho <esteban.jose.garcia.gabancho@cern.ch>
-- Frederic Gobry <frederic.gobry@epfl.ch>
-- giannistsan <tsanakts.10@gmail.com>
-- Glenn Gard <glenng4@aol.com>
-- Gregory Favre <gregory.favre@cern.ch>
-- Guillaume Lastecoueres <PX9e@gmx.fr>
-- Henning Weiler <henning.weiler@cern.ch>
-- Jan Aage Lavik <jan.age.lavik@cern.ch>
-- Javier Martin Montull <javier.martin.montull@cern.ch>
-- Javier Delgado Fernandez <javier.delgado.fernandez@cern.ch>
-- Jerome Caffaro <jerome.caffaro@cern.ch>
-- Jiri Kuncar <jiri.kuncar@cern.ch>
-- Joaquim Rodrigues Silvestre <joaquim.rodrigues.silvestre@cern.ch>
-- Juan Francisco Pereira Corral <Juan.Francisco.Pereira.Corral@cern.ch>
-- Julio Pernia Aznar <jpernia@altransdb.com>
-- kneczaj <kneczaj@gmail.com>
-- Konstantinos Ntemagkos <konstantinos.ntemagkos@cern.ch>
-- Krzysztof Lis <krzysztof.lis@cern.ch>
-- Lars Christian Raae <lars.christian.raae@cern.ch>
-- Lars Holm Nielsen <lars.holm.nielsen@cern.ch>
-- Leonardo Rossi <leonardo.r@cern.ch>
-- Ludmila Marian <ludmila.marian@gmail.com>
-- Marco Neumann <marco@crepererum.net>
-- Marcus Johansson <marcus.johansson@cern.ch>
-- Marko Niinimaki <manzikki@gmail.com>
-- Mathieu Barras <mbarras@gmail.com>
-- Mikael Karlsson <i8myshoes@gmail.com>
-- Nicholas Robinson <nicholas.robinson@cern.ch>
-- Nikolaos Kasioumis <nikolaos.kasioumis@cern.ch>
-- Orestis Melkonian <melkon.or@gmail.com>
-- Patrick Glauner <patrick.oliver.glauner@cern.ch>
-- Paulo Cabral <paulo.cabral@cern.ch>
-- Peter Halliday <phalliday@cornell.edu>
-- Piotr Praczyk <piotr.praczyk@gmail.com>
-- Raquel Jimenez Encinar <raquel.jimenez.encinar@cern.ch>
-- Rémi Ducceschi <remi.ducceschi@gmail.com>
-- Sami Hiltunen <sami.mikael.hiltunen@cern.ch>
-- Samuele Carli <samuele.carli@cern.ch>
-- Samuele Kaplun <samuele.kaplun@cern.ch>
-- Thomas Baron <thomas.baron@cern.ch>
-- Tibor Simko <tibor.simko@cern.ch>
-- Trond Aksel Myklebust <trond.aksel.myklebust@cern.ch>
-- Yoan Blanc <yoan.blanc@cern.ch>
+Contributors
+============
+
+- Adrian Pawel Baran
+- Alexander Ioannidis
+- Alizee Pace
+- Bruno Cuc
+- Chiara Bigarella
+- Diego Rodriguez
+- Dinos Kousidis
+- Eamonn Maguire
+- Esteban J. G. Gabancho
+- Harri Hirvonsalo
+- Harris Tzovanakis
+- Jacopo Notarstefano
+- Javier Delgado
+- Javier Martin Montull
+- Jiri Kuncar
+- Krzysztof Nowak
+- Lars Holm Nielsen
+- Leonardo Rossi
+- Liam Kirsh
+- Nicolas Harraudeau
+- Odd
+- Orestis Melkonian
+- Rémi Ducceschi
+- Sami Hiltunen
+- Sebastian Witowski
+- Tibor Simko
```

Comment:
Contributors issues

Reply:
You mean Odd? Or is there something else missing?

Reply:
It's too long....start commit should be f55f1a909c37cb70d9570ca1d9830269464ad4cd

Reply:
Hmm, from the list you gave me (https://github.com/inveniosoftware/license-changer/blob/master/external_contributors_stats.py#L7), starting commit is f97176828348baeb32bf9556001112ba49bcf4b2

Reply:
Ok, fixed, it was just Adrian who had to be removed.